packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -12,6 +12,8 @@
     keep-anchor-width
     boundary-body
     :stateful="false"
+    anchorSelector=".va-input-wrapper__input"
+    :offset="[0, 2]"
```

Comment:
![image](https://user-images.githubusercontent.com/23530004/146369475-e260181d-9f8c-4b85-917d-a748e5c6af03.png)

Strange offset here.

But according to our design files in figma dropdown should be changed later anyway.

```suggestion
```

Reply:
set 1px offset